package.json

```diff
@@ -23,89 +23,91 @@
   },
   "homepage": "https://github.com/Informasjonsforvaltning/begrepskatalog#readme",
   "dependencies": {
-    "@casl/ability": "^3.2.0",
+    "@a-ignatov-parc/react-resolver": "^3.2.3",
```

Comment:
It looks like a bunch of dev dependencies made it to production ;)

Reply:
commented in utviklingsprat channel